contracts/exchange/contracts/src/MixinSignatureValidator.sol

```diff
@@ -20,24 +20,26 @@ pragma solidity ^0.5.9;
 pragma experimental ABIEncoderV2;
 
 import "@0x/contracts-utils/contracts/src/LibBytes.sol";
+import "@0x/contracts-utils/contracts/src/LibEIP1271.sol";
 import "@0x/contracts-utils/contracts/src/ReentrancyGuard.sol";
-import "@0x/contracts-utils/contracts/src/RichErrors.sol";
 import "@0x/contracts-exchange-libs/contracts/src/LibOrder.sol";
+import "@0x/contracts-exchange-libs/contracts/src/LibZeroExTransaction.sol";
 import "./interfaces/IWallet.sol";
-import "./interfaces/IValidator.sol";
-import "./interfaces/IOrderValidator.sol";
+import "./interfaces/IEIP1271Wallet.sol";
 import "./interfaces/ISignatureValidator.sol";
-import "./MixinTransactions.sol";
 import "./MixinExchangeRichErrors.sol";
+import "./MixinTransactions.sol";
 
 
-contract MixinSignatureValidator is 
-    MixinExchangeRichErrors, 
-    ReentrancyGuard, 
-    LibOrder, 
+contract MixinSignatureValidator is
```

Comment:
This is out of scope for this PR, but I think it could be good to abstract a lot of this logic into a generic library like `libSignatureValidator` that can be used across our packages.